### PR TITLE
Reduce the default slippage from 3% to 2%

### PR DESCRIPTION
**Pull request by dan437**
## Explanation
Reduce the default slippage from 3% to 2%.

## Screenshots
Before:
image

After:
image

### CODE REVIEW

**Review by darkwing — DISMISSED** [review]

**Review by darkwing — DISMISSED** [review]

File: ui/pages/swaps/slippage-buttons/slippage-buttons.js [hunk 1]
```diff
@@ -133,20 +134,20 @@ export default function SlippageButtons({
                       setCustomValue('');
                       setEnteringCustomValue(false);
                       setActiveButtonIndex(0);
-                      onSelect(2);
+                      onSelect(SLIPPAGE.DEFAULT);
                     }}
                   >
-                    2%
+                    {`${SLIPPAGE.DEFAULT}%`}
```
Comment (adonesky1) [inline ui/pages/swaps/slippage-buttons/slippage-buttons.js:140]: Should we localize this? It seems like in[ persian and turkish the percentage sign comes first](https://www.ultius.com/glossary/grammar/punctuation/percentage-sign.html#:~:text=In%20Persian%20and%20Turkish%2C%20the,written%20as%200%2F01).).

File: ui/pages/swaps/slippage-buttons/slippage-buttons.js [hunk 2]
```diff
@@ -30,26 +31,26 @@ export default function SlippageButtons({
   const [customValue, setCustomValue] = useState(() => {
     if (
       typeof currentSlippage === 'number' &&
-      currentSlippage !== 2 &&
-      currentSlippage !== 3
+      currentSlippage !== SLIPPAGE.DEFAULT &&
+      currentSlippage !== SLIPPAGE.HIGH
```
Comment (adonesky1) [inline ui/pages/swaps/slippage-buttons/slippage-buttons.js:35]: [nit]
```suggestion
      !Object.values(SLIPPAGE).includes(currentSlippage)
```

**Review by adonesky1 — DISMISSED** [review]
small nit otherwise LGTM!

File: ui/pages/swaps/slippage-buttons/slippage-buttons.js (hunk 2 above)
Comment (darkwing) [reply ui/pages/swaps/slippage-buttons/slippage-buttons.js]: Ohhh, I like this!

**Review by adonesky1 — APPROVED** [review]

**Review by darkwing — APPROVED** [review]

**Pull request merged**